migration/sql-files/001-audit-logs.sql

```diff
@@ -0,0 +1,25 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- event types
+CREATE TABLE event_type (
```

Comment:
You aren't creating the repository object for this table at this point in time?

Reply:
no, I'm only loading the table with a single "record", which is also in the codebase.